crates/sqlite_store/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+#![doc = include_str!("../README.md")]
+// only enables the `doc_cfg` feature when the `docsrs` configuration attribute is defined
+#![cfg_attr(docsrs, feature(doc_cfg))]
+
+pub mod persist;
+mod schema;
+pub mod store;
+#[cfg(feature = "wallet")]
+#[cfg_attr(docsrs, doc(cfg(feature = "wallet")))]
+pub mod wallet;
+
+use bdk_chain::bitcoin::Network;
+use bdk_chain::{indexed_tx_graph, keychain, local_chain, Anchor, Append};
+use serde::{Deserialize, Serialize};
+
```

Comment:
I think we should publicly re-export the `rusqlite` crate. Therefore the caller can re-use our version instead of explicitly including the dependency in `Cargo.toml`.